test/remote/account_unlock_tests.js

```diff
@@ -186,6 +186,108 @@ TestServer.start(config)
   )
 
   test(
+    'unlock account with incorrect verify code',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'allyourbasearebelongtous'
+      var client
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+      .then(
+        function () {
+          return Client.login(config.publicUrl, email, password)
+        }
+      )
+      .then(
+        function (x) {
+          client = x
+        }
+      )
+      .then(
+        function () {
+          return client.lockAccount(email, password)
+        }
+      )
+      .then(
+        function () {
+          return client.resendAccountUnlockCode()
+        }
+      )
+      .then(
+        function () {
+          return server.mailbox.waitForCode(email)
+        }
+      )
+      .then(
+        function (code) {
+          return client.verifyAccountUnlockCode(client.uid, 'deadbeefbaadf00ddeadbeefbaadf00d')
+        }
+      )
+      .then(
+        function () {
+          t.fail('verifyAccountUnlockCode was expected to fail')
+        },
+        function (error) {
+          t.equal(error.code, 400)
+          t.equal(error.error, 'Bad Request')
+          t.equal(error.errno, 105)
+          t.equal(error.message, 'Invalid verification code')
+        }
+      )
+    }
+  )
+
+  test(
+    'unlock account with bad verify code',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'allyourbasearebelongtous'
+      var client
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+      .then(
+        function () {
+          return Client.login(config.publicUrl, email, password)
+        }
+      )
+      .then(
+        function (x) {
+          client = x
+        }
+      )
+      .then(
+        function () {
+          return client.lockAccount(email, password)
+        }
+      )
+      .then(
+        function () {
+          return client.resendAccountUnlockCode()
+        }
+      )
+      .then(
+        function () {
+          return server.mailbox.waitForCode(email)
+        }
+      )
+      .then(
+        function (code) {
+          return client.verifyAccountUnlockCode(client.uid, 'wibble')
+        }
+      )
+      .then(
+        function () {
+          t.fail('verifyAccountUnlockCode was expected to fail')
+        },
+        function (error) {
+          t.equal(error.code, 400)
+          t.equal(error.error, 'Bad Request')
+          t.equal(error.errno, 107)
+          t.equal(error.message, 'Invalid parameter in request body')
```

Comment:
No, the preceding test handles that case.

I added this one as an extra when I saw it wasn't covered. The linked issue doesn't mention it but I figured it would be okay to go in now anyway.